src/Application.php

```diff
@@ -28,19 +28,6 @@
 class Application extends BaseApplication
 {
     /**
-     * Load all the application configuration and bootstrap logic.
-     *
-     * You can include any other files your application needs to bootstrap
-     * in a web context here.
-     *
-     * @return void
-     */
-    public function bootstrap()
-    {
-        require_once $this->configDir . '/bootstrap.php';
```

Comment:
I don't think changing how bootstraping is done would be commonly needed to warrant having this method in Application class. We could document in manual how bootstraping is done with code example showing how to override this method.